company-profiles/product-hackers.md

```diff
@@ -0,0 +1,33 @@
+# Product Hackers
+
+## Company blurb
+
+Product Hackers is an early company based in Madrid which develop cool software and technology, mainly web and APP. We offer consultancy services, **growth** services -which are our core- and develop in-house products to sell them worldwide.
+
+## Company size
+
+0-20
+
+Actually 15 employees. We plan to grow fast and end 2020 year being 20+ people.
+
+## Remote status
+
+We are a almost-full-time remote company. We normally work on remote all the week and we meet on Mondays in our office, in the center of Madrid.
+Everybody who want to work at our office is always welcome.
+
+## Region
+
+Spain, primarly.
```

Comment:
Is there a reason specifically Spain? Is this employment law or timezones?

Reply:
We still have a physical meeting on Mondays, so we can make teambuilding events. That's the reason we are still hiring only on Spain, actually. Maybe will change soon in future.

Reply:
Let's make this more clear for others who are reading this profile also.

Does everyone working for your company need to be in Madrid for the Monday meetings?  If so:

>Spain, and working once a week from our office in Madrid.

If not, maybe something like this would work:

>Spain only for now, but this may change in the future.